clos.md

```diff
@@ -329,7 +329,13 @@ class (whether or not those instances exist yet).
 
 #### Slot documentation
 
-Each slot accepts one `:documentation` option.
+Each slot accepts one `:documentation` option. To obtain its documentation via `documentation`, you need to obtain the slot object. This can be done compatibly using an implementation such as [closer-mop](https://github.com/pcostanza/closer-mop). For instance:
```

Comment:
> using an implementation

is "implementation" right? I use implementation for SBCL, CCL… I'd use "compatibility library" or "library". Thanks!

Reply:
You're right! Sometimes they speak of MOP implementations, but that's not what closer-mop is. I mixed up the wording.